nox/virtualenv.py

```diff
@@ -52,12 +52,15 @@ def __init__(self, bin=None, env=None):
     def bin(self):
         return self._bin
 
-    def run(self, args, in_venv=True):
+    def run(self, args, in_venv=True, env=None):
         """Runs a command. By default, the command runs within the
         environment."""
+        env_copy = self.env.copy() if in_venv else None
```

Comment:
So this causes a weird situation where if you pass both `in_venv=False` and `env={...}` then `env` won't be applied.

Recommendation:

```python
command_env = {}
if in_venv:
   command_env.update(self.env)
```

Reply:
I thought about that use case too, but since `env` was only being passed down when `self.in_venv = true` I assumed that you were only able to use `env`s within a `venv` for some reason. Applying the suggested patch right now 👍